### PR TITLE
chore(modules): update some modules to latest version

### PR DESCRIPTION
**Pull request by Lctrs**

### CODE REVIEW

**Comment by gasinvein** [comment]
Nitpick: updating modules seems like `chore` to me, not `feat`.

**Comment by Lctrs** [comment]
Yeah I think you're right. But I don't think it's worth retriggering a build for this.

File: modules-32bit.yml [hunk 1]
```diff
@@ -55,8 +55,6 @@ modules:
       arch:
         x86_64: *compat_i386_opts
     buildsystem: cmake-ninja
-    config-opts:
-      - -DGSTREAMER=ON
```
Comment (nanonyme) [inline modules-32bit.yml:59]: Why was this removed?

Comment (Lctrs) [reply modules-32bit.yml:59]: Because [GStreamer support has been removed in FAudio 22.02](https://github.com/FNA-XNA/FAudio/releases/tag/22.02).

Comment (Lctrs) [reply modules-32bit.yml:59]: (and therefore, the option doesn't exist anymore)

Comment (gasinvein) [reply modules-32bit.yml:59]: Ugh. I wonder why we need FAudio at all now, then.

Comment (Lctrs) [reply modules-32bit.yml:59]: I see now that I have missed the i386 counterpart.

Comment (nanonyme) [reply modules-32bit.yml:59]: Maybe we don't? It would be great to remove it, and the s-l-g overrides.

Comment (Lctrs) [reply modules-32bit.yml:59]: I'll remove the relevant commit from this PR and make a new one for it.

Comment (gasinvein) [reply modules-32bit.yml:59]: AFAIK it was needed exactly because it provided proprietary codecs support for Proton. If it doesn't provide it anymore, I'm not sure what else it's useful for.

**Comment by Lctrs** [comment]
@nanonyme @gasinvein Here it is without the FAudio update

**Comment by nanonyme** [comment]
fedc is fine if it works here.

File: modules-32bit.yml (hunk 1 above)
Comment (nanonyme) [reply modules-32bit.yml]: it was removed through second PR so I'm resolving this.
